cumulusci.yml

```diff
@@ -244,13 +252,19 @@ flows:
                 task: deploy_customer_profiles
                 options:
                     unmanaged: False
+            5:
```

Comment:
In a managed install, you're going to get the packaged flows. Do you want to install the unmanaged ones too?

Reply:
@davidmreed , Nope we do not want to install the unmanaged one's. I have gone ahead and removed them.